src/status_im2/contexts/share/style.cljs

```diff
@@ -0,0 +1,99 @@
+(ns status-im2.contexts.share.style
+  (:require [quo2.foundations.colors :as colors]))
+
+(def screen-padding 20)
+
+(def header-button
+  {:margin-bottom 12
+   :margin-left   screen-padding})
+
+(def header-heading
+  {:padding-horizontal screen-padding
+   :padding-vertical   12
+   :color              colors/white})
+
+(defn screen-container
+  [window-width top bottom]
+  {:flex           1
+   :width          window-width
```

Comment:
`:width          window-width` why this needed?

Reply:
we need to set this in the event the container has no content, for example when you switch to wallet tab since there is no width the view looks like this if I remove this code : 

<img width="534" alt="Screenshot 2023-04-04 at 11 40 11 AM" src="https://user-images.githubusercontent.com/64726664/229703039-f0a67cdd-c7a3-4547-9c12-244765460137.png">

Reply:
seems this use of window-width is still here? this should be adjusted too if possible 👍

Reply:
This is just for the wallet-tab at the moment, it will go away once that comes in